apps/ui/src/components/Modal/Transaction.vue

```diff
@@ -164,13 +164,13 @@ async function handleToChange(to: string) {
     return;
   }
 
-  if (typeof props.network === 'string') {
+  if (getChainIdKind(props.network) === 'starknet') {
```

Comment:
I'd flip those conditions (`!== 'evm'`) so in the future when new network kind is introduced it won't break. With that we can update message to say "only evm is supported" instead of "starknet is not supported").